lib/game.rb

```diff
@@ -3,12 +3,19 @@
 class Game
   attr_reader :board
   def initialize
-    @board = [0, ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+    @board = [' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ', ' ']
+    @arr = []
   end
 
   def play(position, tool)
     @position = position
     @tool = tool
-    @board[@position] = @tool
+    @board[@position - 1] = @tool
+  end
```

Comment:
All the public method from Classes should be tested, but there is not a test method for this one.
Please, could you write it test method ?